### DISCUSSION TITLE
arm64 manual installation size

### DISCUSSION THREAD

**Discussion by demetera** [TriliumNext/Trilium#3552] [Q&A]
Hi all,

after installing self hosted server manually, using lxc container - git folder size is around 600 Mb : node_modules takes around 95%. After comparing with amd64 installation noticed significant difference.
Any ideas how to reduce the size of the folder? I'm not sure everything is required (size almost like for debian distro)

In doc it's only : cd trilium - > npm installl  -> npm rebuild or I've missed anything?

Thanks

**Comment by zadam** [comment]
You can delete the .git folder (upgrading might be a pain though). There's unnecessary stuff in node_modules, but it's a pain to identify what exactly. It's recommended to use a docker image instead - 111 MB for ARM64 https://hub.docker.com/r/zadam/trilium/tags

**Reply by demetera** [reply]
Thanks for your reply. Definitely yes, ended up with docker image on lxc container and this is the best choice for portability and experiments
